h/static/images/icons/account.svg

```diff
@@ -5,9 +5,9 @@
     <desc>Created with Sketch.</desc>
     <defs></defs>
     <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
-        <g id="Artboard-1-Copy" sketch:type="MSArtboardGroup" fill="#000000">
+        <g id="Artboard-1-Copy" sketch:type="MSArtboardGroup" fill="currentColor">
```

Comment:
Does `fill="currentColor` enable the colour of the user icon to be changed by CSS? I was fiddling with `color` and `fill` CSS classes on some of the elements in dev tools but couldn't seem to get it to change